web/src/components/apps/AppDetailPage.jsx

```diff
@@ -47,6 +47,7 @@ class AppDetailPage extends Component {
       displayErrorModal: false,
       isVeleroInstalled: false,
       redeployVersionErrMsg: "",
+      checkForFirstAppJob: new Repeater(),
```

Comment:
don't like using this but will swap out to use react-query when we consolidate apis. react-query has this functionality built in